GoogleMulticastAppDelegate/Sources/MulticastAppDelegate.swift

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Google LLC
```

Comment:
As nice as it would be, I don't think we can use Swift in libraries in Analytics dependency chain since SPM does not support Swift source libraries as dependencies of binary libs. See https://groups.google.com/a/google.com/g/google-ios-sdks/c/XsapHN-58Tk/m/dIEEnmpaAwAJ?utm_medium=email&utm_source=footer

Reply:
Ah, I didn't know about it. It's a bit disappointing. So far it looks like we can re-implement the multicast delegate in Objective-C though it will be more verbose and require Swift name annotations for some methods (which is still much easier to do comparing to the swizzling version). Thank you for the heads up!

Reply:
@paulb777 @maksymmalyhin IIRC, there was a discussion about whether this is possible or not but I forgot the outcome. Does SPM  support Swift source libraries as dependencies of binary libs?

Reply:
We have not detected any issues with it.

Reply:
There is a problem with using Swift source dependencies from Swift binary pods, but we don't plan to have any Swift binary pods.